src/Endpoints/README.md

```diff
@@ -0,0 +1 @@
+# wp-parsely Plugin WordPress REST API Endpoints
```

Comment:
I see that this documentation will be handled in a separate ticket. If so, I'd wouldn't include this README file (it's essentially empty) and add it later when it has content.